src/pages/projectSettings/tabs/NotificationsTab/SubscriptionField.tsx

```diff
@@ -10,6 +10,19 @@ import { ResourceType, StringMap, Trigger } from "types/triggers";
 import { toSentenceCase } from "utils/string";
 import { SubscriptionSelect } from "./NotificationsRow";
 
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+const generateWebhookSecret = () => {
```

Comment:
We may want to generate this in evergreen if it's private information or if we want to guarantee unique keys

Reply:
This is ready for re-review. Please see the comment below.